### DISCUSSION TITLE
Toolbar and an inventory to store chips

### DISCUSSION THREAD

**Discussion by GeoffreyParrier** [UkrainianBanderasCat/Digital-Logic-Sim2#64] [Ideas]
How about an inventory and a toolbar to chose which we used, with drag and drop of item.

After a little while I did something that can permit to manage more easily chips and other things (and access edit and delete button without reworking on the whole interface already implemented).

Here's an example of what could be done : 
![image](https://user-images.githubusercontent.com/47033480/179421561-02c45dbc-b607-42a2-b4fa-e3171c5552bc.png)
![image](https://user-images.githubusercontent.com/47033480/179421567-8d9307f4-733c-407a-8b20-d694dcaf1eca.png)
![image](https://user-images.githubusercontent.com/47033480/179421577-ac860e79-a3ce-4041-b037-168d547b3e72.png)

**Comment by UkrainianBanderasCat** [comment]
Looks cool
